app/src/main/java/kr/co/softcampus/sopt_assignment1/HomeActivity.kt

```diff
@@ -14,10 +15,20 @@ class HomeActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        binding = ActivityHomeBinding.inflate(layoutInflater)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_home)
+
+        val intro = Introduce(
+            "최유림",
+            24,
+            "esfp",
+            "안녕하세요 SOPT 29기 \n안드로이드 YB입니다\n잘 부탁드립니다~♥\n\n안녕하세요 SOPT 29기 \n안드로이드 YB입니다\n잘 부탁드립니다~♥\n\n안녕하세요 SOPT 29기 \n안드로이드 YB입니다\n잘 부탁드립니다~♥\n\n안녕하세요 SOPT 29기 \n안드로이드 YB입니다\n잘 부탁드립니다~♥\n\n안녕하세요 SOPT 29기 \n안드로이드 YB입니다\n잘 부탁드립니다~♥",
+            R.drawable.lim
+        )
+
+        binding.introduce = intro
+
         binding.git.setOnClickListener {
-            val url = "https://github.com/ChoiYuLim"
-            val webPage: Uri = Uri.parse(url)
+            val webPage: Uri = Uri.parse("https://github.com/ChoiYuLim")
```

Comment:
코틀린의 타입추론 기능을 사용해서 이렇게도 표현가능합니다 :)
```suggestion
            val webPage = Uri.parse("https://github.com/ChoiYuLim")
```